app/renderer/components/Inspector/RecordedActions.js

```diff
@@ -89,7 +89,7 @@ class RecordedActions extends Component {
 
     const highlightedCode = this.code(false);
 
-    return <Card title={<span><Icon type="code-o"/>{t('Recorder')}</span>}
+    return <Card title={<span><Icon type="code-o"/> {t('Recorder')}</span>}
```

Comment:
is that space needed?

Reply:
It is, yeah. It looks weird if there's no space between it and the icon.